app/src/main/res/layout/item_timeline.xml

```diff
@@ -34,27 +34,33 @@
             android:gravity="start"
             android:orientation="vertical">
 
-            <RelativeLayout
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content">
                 <TextView
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:id="@+id/text_timeline_date"
-                    android:textSize="12sp"
-                    tools:text="2018-1-1 12:00"
-                    android:layout_alignParentLeft="true"
-                    android:layout_alignParentStart="true"/>
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:layout_weight="1"
```

Comment:
Just for info, I think layout_weight is worth nothing if you don't put layout_height = 0 in case of vertical LinearLayout and layout_width = 0 in case of horizontal LineaLayout

Reply:
It allows for all items to have equal weight in the line and therefore to share the space equally

Reply:
Yes but in case of a horizontal LinearLayout, the layout_weight will be overriden by the value in layout_width if it is not 0, same goes accordingly for vertical and layout_height.

Reply:
Ok ! Thanks for explanations !